types/react-native/index.d.ts

```diff
@@ -1075,10 +1090,13 @@ export interface TextInputIOSProps {
     | 'newPassword'
     | 'oneTimeCode';
 
+
```

Comment:
Unnecessary blank lines?

Reply:
Yup, this is causing CI to fail.

Reply:
oops, I don't know how that snuck in there 😅 
I've removed the blank lines, thanks!